### DISCUSSION TITLE
renovate-runner (GitLab CI) how to direct all logs to stdout

### DISCUSSION THREAD

**Discussion by stefandoorn** [renovatebot/renovate#16445] [Request Help] [answered]
I'm using `renovate-runner` on self-hosted GitLab CI runners. The log file is saved as an artefact, which I can download and pipe through bunyan. I would like to have the log output, in readable format, just piped into `stdout` which shows into the GitLab CI pipeline run. All projects are internal, and then it's easy to debug things.

Is there a way to either keep the log file & also output it to `stdout`, or just only to `stdout`?

**Comment by rarkins** [comment, marked as answer]
Have you seen LOG_FILE and LOG_FORMAT options? First let's you log to both a file and stdout. Second let's you change stdout formatting to be line separated JSON

**Reply by MaronHatoum** [reply]
please see [logging documentation](https://docs.renovatebot.com/examples/self-hosting/#logging).

**Comment by stefandoorn** [comment]
Thanks!
